src/main/java/com/spartronics4915/frc2020/subsystems/Launcher.java

```diff
@@ -0,0 +1,76 @@
+package com.spartronics4915.frc2020.subsystems;
+
+import com.spartronics4915.frc2020.Constants;
+import com.spartronics4915.lib.hardware.motors.SpartronicsSRX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.spartronics4915.lib.subsystems.SpartronicsSubsystem;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+public class Launcher extends SpartronicsSubsystem {
+    CANSparkMax mFlywheelMasterMotor;
+    CANSparkMax mFlywheelFollowerMotor;
+    TalonSRX mAngleAdjusterMotor;
+    TalonSRX mTurretMotor;
+    public Launcher() {
+        // Construct your hardware here
+        logInitialized(true);
```

Comment:
You don't want this `logInitialized` here.

Something I was planning to teach later, but that you can do right now, is wrapping our initializations in a try / catch loop so that we can log the error to the console. [See here for an example (pretend the comments aren't commented out)](https://github.com/Spartronics4915/2017-STEAMworks/blob/command-rewrite-2020/src/main/java/com/spartronics4915/frc2017/subsystems/Launcher.java#L25).

Reply:
:+1: